src/Spago/Build.hs

```diff
@@ -131,7 +131,7 @@ repl cacheFlag newPackages sourcePaths pursArgs depsOnly = do
       Temp.withTempDirectory cacheDir "spago-repl-tmp" $ \dir -> do
         Turtle.cd (Turtle.decodeString dir)
 
-        Packages.initProject False
+        Packages.initProject False False
```

Comment:
Not sure if this default is ok here..

Reply:
It is, but since we're starting to add booleans in here let's make some datatypes instead to avoid confusion (it's ok if you don't refactor the rest and only add a `data TemplateComments = WithComments | NoComments`)

Reply:
Absolutely, I agree. I didn't like another Boolean flag from the beginning (boolean blindness and stuff y'know), but wasn't sure where to put that data type. I added it to `Spago.Dhall` in my next commit. This is I think the most logical place for it (together with comment stripping functionality).

Reply:
This works well, thanks! I think we'll eventually want to move all types to the `Spago.Types` module so that they are not scattered around the codebase, but on the other hand it's also OK to have them where they provide context